### ISSUE TITLE
Bug(preload-langs): NullInjectorError: No provider for TranslocoPreloadLangsService | Angular 16

### ISSUE THREAD

**Issue by baikov-ilia** [ngneat/transloco#726]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Transloco package(s) are the source of the bug?

Transloco, Preload Langs

### Is this a regression?

Yes

### Current behavior

```
NullInjectorError: R3InjectorError(AppModule)[ApplicationInitStatus -> InjectionToken Application Initializer -> [object Object] -> TranslocoPreloadLangsService -> TranslocoPreloadLangsService]: 
  NullInjectorError: No provider for TranslocoPreloadLangsService!
    at NullInjector.get (core.mjs:8890:27)
    at R3Injector.get (core.mjs:9334:33)
    at R3Injector.get (core.mjs:9334:33)
    at injectInjectorOnly (core.mjs:842:40)
    at ɵɵinject (core.mjs:848:60)
    at injectArgs (core.mjs:977:23)
    at Object.factory (core.mjs:9553:52)
    at core.mjs:9454:43
    at runInInjectorProfilerContext (core.mjs:798:9)
    at R3Injector.hydrate (core.mjs:9453:17)
    ```

### Expected behavior

Preload languages and scopes are loaded.

### Please provide a link to a minimal reproduction of the bug, if you won't provide a link the issue won't be handled.

https://codesandbox.io/s/ngneat-transloco-forked-9qvfnk

### Transloco Config

```markdown
@NgModule({
  exports: [ TranslocoModule ],
  providers: [
      provideTransloco({
        config: {
          availableLangs: ['ru', 'en'],
          defaultLang: 'ru',
          // Remove this option if your application doesn't support changing language in runtime.
          reRenderOnLangChange: true,
          prodMode: !isDevMode(),
        },
        loader: TranslocoHttpLoader
      }),
      provideTranslocoPersistLang({
        storage: {
          useValue: cookiesStorage()
        }
      }),
    provideTranslocoPreloadLangs(['ru', 'en']),
  ],
})
export class TranslocoRootModule {}
```

### Please provide the environment you discovered this bug in

```markdown
Transloco: ^6.0.0 (^5.0.0 as well)
Angular: ^16.2
Node: 18.17.1
Package Manager: npm
OS: Win 11 x64
```

### I would like to make a pull request for this bug

No

**Comment by EMenardc** [comment]
Same error with angular v17

**Comment by shaharkazaz** [comment]
@EMenardc @baikov-ilia 
Released v5.0.1, thank you @Owain94.
Please LMK if the issue is resolved on your end.

**Comment by baikov-ilia** [comment]
> @EMenardc @baikov-ilia 
> 
> Released v5.0.1, thank you @Owain94.
> 
> Please LMK if the issue is resolved on your end.

Yep, it's been resolved. Thank you
